llmware/library.py

```diff
@@ -483,7 +685,30 @@ def add_files (self, input_folder_path=None, encoding="utf-8",chunk_size=400,
     def export_library_to_txt_file(self, output_fp=None, output_fn=None, include_text=True, include_tables=True,
                                    include_images=False):
 
-        """ Exports library collection of indexed text chunks to a txt file """
+        """ Exports library collection of indexed text chunks to a txt file 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Exports library collection of indexed text chunks to a txt file.
```